input/pagecontent/conformance-patients.md

```diff
@@ -1,14 +1,14 @@
-To facilitate conformance testing, testing software must be able to determine which patients are "mCODE Patients" -- those in scope for mCODE. In general, all patients with confirmed cancer diagnoses SHOULD be covered by mCODE. In FHIR terms, these are patients who have a Condition where `Condition.code` is a member of the value set [PrimaryOrUncertainBehaviorCancerDisorderVS] and `Condition.verificationStatus` is confirmed.
+To facilitate conformance testing, testing software must be able to determine which patients are in-scope -- Cancer patients who are in the scope of mCODE. In general, all patients with confirmed cancer diagnoses SHOULD be in-scope. In FHIR terms, these are patients who have a Condition where `Condition.code` is a member of the value set [PrimaryOrUncertainBehaviorCancerDisorderVS] and `Condition.verificationStatus` is confirmed.
```

Comment:
```suggestion
To facilitate conformance testing, testing software must be able to determine which patients are in-scope (meaning cancer patients who are in the scope of mCODE for conformance purposes). In general, all patients with confirmed cancer diagnoses SHOULD be in-scope. In FHIR terms, these are patients who have a Condition where `Condition.code` is a member of the value set [PrimaryOrUncertainBehaviorCancerDisorderVS] and `Condition.verificationStatus` is confirmed.
```

Reply:
Similar comment to one above -- do not define in-scope using the words "in-scope"